src/vs/workbench/contrib/url/common/trustedDomains.ts

```diff
@@ -51,9 +51,10 @@ export function normalizeURL(url: string | URI): string {
 	} catch { return url.toString(); }
 }
 const rLocalhost = /^localhost(:\d+)?$/i;
+const rLocalhostTld = /^(.+\.)?localhost(:\d+)?$/i;
```

Comment:
why not just update the `rLocalhost` one?

Reply:
Fair point, was trying to be explicit, but not needed.